lib/middleware/timeouts.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const timeout = require('connect-timeout');
+
+// Default limit in seconds before requests timeout
+const DEFAULT_TIMEOUT_NUM_SECONDS = 15;
+
+/**
+ * Returns number of seconds to use for Gambit timeout setting.
+ * @return {number}
+ */
+module.exports.getGambitTimeoutNumSeconds = function () {
+  return Number(process.env.GAMBIT_TIMEOUT_NUM_SECONDS) || DEFAULT_TIMEOUT_NUM_SECONDS;
```

Comment:
Minor question -- would it be cleaner to just define our backup as 15 here, vs earlier in the file?